src/lib/autoinstall/widgets/storage/lvm_page.rb

```diff
@@ -19,8 +19,7 @@
 
 require "yast"
 require "y2storage"
-require "cwm/page"
-require "autoinstall/widgets/storage/add_children_button"
+require "autoinstall/widgets/storage/drive_page"
 require "autoinstall/widgets/storage/vg_device"
 require "autoinstall/widgets/storage/vg_extent_size"
 require "autoinstall/widgets/storage/md_level"
```

Comment:
I forgot to remove those three `require`s which are used by the _raid_page_ Maybe we can delete them as part of this PR

Reply:
Done